AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -51,9 +55,11 @@ public final class QRCodeComponent: ActionComponent, Localizable, Cancellable {
     /// - Parameter style: The component UI style.
     /// - Parameter pollingComponentBuilder: The payment method specific await action handler provider.
     /// - Parameter timeoutInterval: QR Code expiration timeout
-    internal init(style: QRCodeComponentStyle = QRCodeComponentStyle(),
+    internal init(apiContext: APIContext,
+                  style: QRCodeComponentStyle = QRCodeComponentStyle(),
                   pollingComponentBuilder: AnyPollingHandlerProvider?,
```

Comment:
This class can probably build it's own `PollingComponent`

Reply:
maybe we could do this in another PR